jsapp/js/components/formGallery/formGallery.selectors.ts

```diff
@@ -100,9 +100,15 @@ export const selectFilterQuery = (
     // $and is necessary as repeating a json key is not valid
     const andQuery: Json = [];
     if (startDate) {
+      if (startDate.includes('T') == false) {
+        startDate = startDate + 'T00:00Z';
+      }
```

Comment:
@p2edwards, would you leave this as-is, or prefer it to use `===` or be `if (!startDate.includes('T'))`?